### ISSUE TITLE
Pros against native galera support in proxysql

### ISSUE THREAD

**Issue by balbaev** [Tusamarco/proxy_sql_tools#23]
I've tested your scheduler and tried to compare it with native from proxysql, but it's not fully clear for me who's better. Can you help me to make a decision please?

**Comment by balbaev** [comment]
I've found the [article](https://www.percona.com/blog/2020/11/30/support-for-percona-xtradb-cluster-in-proxysql-part-two/) where Marco warns that his script is not for production use.

**Comment by Tusamarco** [comment]
Are you the same person who raise the question in percona channel? Of so is
the reply enough if not let me know and I will give you my full reply

**Comment by balbaev** [comment]
Yes, I'm the same person from the channel. In my case your reply was enough, but it would be nice to have your full answer, probably I missed smth and it would be handy for newcomers.
